frontend/src/components/IntegrationConfiguration.module.scss

```diff
@@ -56,7 +56,6 @@
 
 .mappings {
   @extend .layout-column;
-  gap: 20px;
```

Comment:
I think some gap would be good here.

Now the rows are right next to each other, but there is 5px margin on the top of the dropdown button which makes the small visual gap

Reply:
I liked how it looked like without the gap, but I added a 5px gap.